### ISSUE TITLE
[Bug]: It's not clear that the 'Pocket' toggle is for 'Through provoking stories'

### ISSUE THREAD

**Issue by jonalmeida** [mozilla-mobile/fenix#21787]
When trying to turn features on and off the 'Homepage' settings, it isn't clear what "Pocket" is, or vice versa, it isn't clear that 'Thought provoking stories' on the homescreen is Pocket. See below for my high-quality ASCII figma designs:

## Expected results

The title is similar in settings:
```
Most visited topsites                [ ]

Jump back in                         [x]

Recent bookmarks                     [ ] 

Recent searches                      [x]

Thought provoking stories            [x]
_powered by Pocket_
```

## Actual results
```
Most visited topsites                [ ]

Jump back in                         [x]

Recent bookmarks                     [ ] 

Recent searches                      [x]

Pocket                               [x]

```

<img width="878" alt="out" src="https://user-images.githubusercontent.com/1370580/136470623-03583f44-a6a4-46bd-a9a4-15d42dda55bb.png">

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/FNXV2-21098)

**Comment by AdinaPetridean** [comment]
Verified as fixed on Nightly 108.0a1 from 10/20 with Xiaomi Pad5 (Android 12). The 'Thought provoking stories' (Articles powered by Pocket) toggle label is displayed in Homepage  - Settings instead of 'Pocket'.

!['Thought provoking stories'](https://user-images.githubusercontent.com/108326843/196907094-99d7f973-a1e7-4ca2-8ce2-44b2cb27db54.JPEG)

**Comment by Alexandru2909** [comment]
Figma designs were provided in the Jira ticket synchronized with this issue.
